_posts/en/newsletters/2022-06-22-newsletter.md

```diff
@@ -0,0 +1,230 @@
+  many free and open source software Bitcoin projects).
+
+- **Timestamping design:** A protracted debate on the Bitcoin-Dev
+  mailing list about the design of the Bitcoin-based [Open Timestamps][]
+  (OTS) system seemed to [conclude][poelstra timestamping] this week.
+  The source of the debate appears to have been the existence of two
+  different designs for time stamping systems:
+
+    - *Time Stamped Proofs of Existence (TSPoE):* a Bitcoin transaction
+      commits to a hash digest which commits to a document.  When the
+      transaction is confirmed in a block, it's possible for the creator
+      of the commitment to prove to third parties that the document
+      existed at the time the block was created.  Notably, each time
+      stamping transaction can be completely independent from other time
+      stamping transactions, meaning it's possible to timestamp the same
+      document multiple times with no connection between the time
+      stamps.
+
+    - *Event Ordering (EO):* a series of transactions all related to
+      each other in a specified manner each commits to documents in a
+      way that allows any user of the system to see all the commitments.
+      For any document that is timestamped two or more times under this
+      system, it is possible to determine when it was first timestamped.
+
+  The TSPoE system as implemented by OTS is essentially perfectly
+  efficient.  It uses the same amount of global storage space to time
+  stamp an unlimited number of documents, with each person who requests
+  a timestamp being responsible for storing their time stamp proofs.
+  This system also has the advantage of being simple both in concept and
+  implementation.
+
+  The EO system requires all full participants store the commitments to
+  every document.  This can be much less efficient and adds complexity.
+  The tradeoff is that it does allow participants to verify when a
+  document was first published to the system.
+
+  The discussion did not lead to any announced changes in any system or
+  proposal, such as Open Timestamps or transaction sponsorship (the
+  original topic of discussion, see [Newsletter #116][news116
+  sponsorship]).  It did seem to surprise several discussion
+  participants that they could each have different concepts of what
+  "time stamping" implied.
+
+- **New RIDDLE anti-sybil method:** Adam "Waxwing" Gibson
+  [posted][gibson riddle post] to the Bitcoin-Dev mailing list a
+  [proposal][gibson riddle gist] for an [anti-sybil][sybil] mechanism
+  that uses the Bitcoin UTXO set and which can provide reasonably good
+  privacy.  A user can generate a list of UTXOs where one of the UTXOs
+  belongs to the user and the rest belong to other users.  The user then
+  creates a signature which third parties can verify came from an owner
+  of one of the UTXOs---but which the verifier can't determine which of
+  the listed UTXO owners created it.
```

Comment:
Too many witches spoil the potion—that last sentence is a bit hard to parse. Perhaps:

```suggestion
  belongs to the user and the rest belong to other users.  The user then
  creates a signature which provably came from an owner
  of one of the listed UTXOs, but does not reveal which owner created it.
  ```